### PR TITLE
added sortable tabs 

### PR DESCRIPTION
**Pull request by jgeddes-coveo**

### CODE REVIEW

**Comment by jgeddes-coveo** [comment]
see the code sandbox here: https://codesandbox.io/s/nameless-glitter-r9s6q?file=/src/Components/Tab.tsx

File: src/App.tsx [hunk 1]
```diff
@@ -24,6 +26,18 @@ export default class App extends React.Component {
           <Typography variant="h4" component="h1" gutterBottom>
             Coveo Headless + Material UI
           </Typography>
+          <CenteredTabs>
+            <HeadlessTab label="All Content" expression="" />
+            <HeadlessTab
+              label="Countries"
+              expression='@source=="	
+              Coveo Sample - ListCountries"'
```
[the remark this answers is not in the document]
Comment (kaitlynkatz) [reply src/App.tsx:34]: Codesandbox automatically formats the code for you when you save, so I'm pretty sure this is just bc the source name is long

File: src/Components/Tab.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-use-before-define */
+import React from "react";
+import { Tab as MaterialUITab, TabProps } from "@material-ui/core/";
+import { buildTab, Tab, TabState } from "@coveo/headless";
+import { headlessEngine } from "../Engine";
+
+interface ITabProps extends TabProps {
+  expression: string;
+}
+
```
[the remark this answers is not in the document]
Comment (kaitlynkatz) [reply src/Components/Tab.tsx:10]: We should definitely get the headless team to take a look- I can't tell from the source code how to do this

[the remark this answers is not in the document]
Comment (jpmarceau) [reply src/Components/Tab.tsx:10]: I agree. Also, I checked and there's no "tab" in the search request.

I don't really have anything else to say on the PR, so I'd say this is a good time to add some folds form the ui-kit team to the review 👍

[the remark this answers is not in the document]
Comment (jgeddes-coveo) [reply src/Components/Tab.tsx:10]: @jpmarceau do you suggest anyone in particular?

[the remark this answers is not in the document]
Comment (jpmarceau) [reply src/Components/Tab.tsx:10]: You can add Olivier Lamothe, Sami Sayegh and Jean-Félix Thibodeau

[the remark this answers is not in the document]
Comment (samisayegh) [reply src/Components/Tab.tsx:10]: Good points, I have logged this jira to fix the two issues: https://coveord.atlassian.net/browse/KIT-319

**Review by samisayegh — APPROVED** [review]

**Review by ThibodeauJF — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -5,7 +5,7 @@
   "keywords": [],
   "main": "src/index.tsx",
   "dependencies": {
-    "@coveo/headless": "0.1.0-alpha.3",
+    "@coveo/headless": "0.1.0-alpha.29",
```
[the remark this answers is not in the document]
Comment (jpmarceau) [reply package.json:8]: Yep, we started this task on version 29. We'll keep that sandbox version for that version of the documentation.
In any case we'll want to create a new version and update as needed for the release this Friday.

**Review by jpmarceau — APPROVED** [review]
Nice work @jgeddes-coveo and @kaitlynkatz !

File: src/Components/CenteredTabs.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    this.setState({ value: newValue });
+  };
+
+  render() {
+    return (
+      <Paper>
+        <Tabs
+          value={this.state.value}
+          onChange={this.handleChange}
+          indicatorColor="primary"
```
[the remark this answers is not in the document]
Comment (dmbrooke) [reply src/Components/CenteredTabs.tsx:20]: I personally find the orange to be much easier to read :) You're right, we'll need to keep an eye out for the future rebranding, but I think readability should take priority over all else (it wouldn't surprise me if at least some of our readers had varying degrees of colour blindness)

**Review by dmbrooke — APPROVED** [review]
Didn't really have anything new to add to the conversation (sorry for such a late review), but this looks great! Nice work :)

**Pull request merged**